set.mm

```diff
@@ -171874,19 +172004,53 @@ seq m ( + , ( n e. ZZ |-> if ( n e. A , [_ n / k ]_ B , 0 ) ) ) ~~> x ) \/
   $}
 
   ${
-    $d A k x $.  $d B x $.  $d k x z $.
+    $d A k x $.  $d B x $.  $d Z k x $.
     $( A finite sum with an additional integer summand is an integer.
-       (Contributed by Alexander van der Vekens, 1-Sep-2018.) $)
-    fsummsnunz $p |- ( ( A e. Fin /\ A. k e. ( A u. { z } ) B e. ZZ )
+       (Contributed by Alexander van der Vekens, 1-Sep-2018.)  (Revised by AV,
+       17-Dec-2021.) $)
+    fsummsnunz $p |- ( ( A e. Fin /\ A. k e. ( A u. { Z } ) B e. ZZ )
+                       -> sum_ k e. ( A u. { Z } ) B e. ZZ ) $=
+      ( vx cfn wcel cz csn cun wral wa csu cv nfcv nfcsb1v csbeq1a cbvsumi snfi
+      csb a1i unfi syldan wi rspcsbela expcom adantl imp fsumzcl syl5eqel ) AFG
+      ZBHGCADIZJZKZLZUMBCMUMCENZBTZEMHUMBUQCEEBOCUPBPCUPBQRUOUMUQEUKUNULFGZUMFG
+      URUODSUAAULUBUCUOUPUMGZUQHGZUNUSUTUDUKUSUNUTCUPUMBHUEUFUGUHUIUJ $.
+
+    $d V x $.
+    $( Separate out a term in a finite sum by splitting the sum into two parts.
```

Comment:
Single out...

Reply:
"Separate out" is often used (by many contributors) and in similar cases, whereas "single out" is only used once (in a different case) in set.mm. Therefore, I would not change this.